.github/workflows/release_to_azure.yml

```diff
@@ -144,7 +144,10 @@ jobs:
         env:
           checksum_validation: ${{ vars.CHECKSUM_VALIDATION }}
         if: needs.pre_job.outputs.has_router_change == 'true' && env.checksum_validation == 'true'
+
         uses: JosiahSiegel/checksum-validate-action@ebdf8c12c00912d18de93c483b935d51582f9236
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/checksum-validate-action
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.